app/assets/javascripts/modules/confirm.js

```diff
@@ -2,6 +2,7 @@
 
 moj.Modules.jsConfirm = {
   confirmClass: 'js-confirm',
+  confirmMessage: 'Are you sure?',
```

Comment:
Not important, but should this message be a bit more specific, like "Are you sure you want to delete this thing?"

Reply:
Could be. Or, actually, it could be moved out of the JS and into a `data-` attribute so that the module can be used on any confirmable button/action.